File: backend/components/metiers/administration_publique/utils/validators/helpers/validators_helper.test.js
// validators_helper.test.js
// Tests unitaires JS pour validators_helper
const { isValidEmail } = require('./validators_helper');

describe('isValidEmail', () => {
  it('valide un email correct', () => {
    expect(isValidEmail('test@example.com')).toBe(true);
  });
  it('rejette un email incorrect', () => {
    expect(isValidEmail('test@.com')).toBe(false);
    expect(isValidEmail('test.com')).toBe(false);
  });
});

File: backend/components/metiers/administration_publique/utils/validators/helpers/validators_helper.test.py
# validators_helper.test.py
# Tests unitaires Python pour validators_helper
from .validators_helper import is_valid_email

def test_is_valid_email():
    assert is_valid_email('test@example.com')
    assert not is_valid_email('test@.com')
    assert not is_valid_email('test.com')
